packages/components/app/styles/components/application-state.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APPLICATION-STATE COMPONENT
+//
+$hds-application-state-padding: 12px 0;
```

Comment:
Would you want to create a token for the vertical padding value?

Reply:
I don't have strong opinions; it seemed consistent with other components.

Reply:
It seems like we normally tokenize this type of value to make it easier for consumers to override if they wish to.